packages/material-ui-styles/src/makeStyles/makeStyles.d.ts

```diff
@@ -69,7 +69,7 @@ export type StylesHook<S extends Styles<any, any>> = StylesRequireProps<S> exten
   ? (props?: any) => ClassNameMap<ClassKeyOfStyles<S>>
   : (props: PropsOfStyles<S>) => ClassNameMap<ClassKeyOfStyles<S>>;
 
-export default function makeStyles<S extends Styles<any, any>>(
+export default function makeStyles<S extends Styles<Theme, any>, Theme>(
```

Comment:
@vitkon I have a better solution. Pushing to your branch.

Reply:
thanks